src/dotnet-interactive-vscode/insiders/package.json

```diff
@@ -55,7 +55,8 @@
   ],
   "capabilities": {
     "untrustedWorkspaces": {
-      "supported": true
+      "supported": "limited",
+      "description": "Execution of cells in Interactive Window and Notebooks is not supported in untrusted workspaces."
```

Comment:
Same change is needed in stable `package.json`, unless untrusted workspaces isn't a thing in stable.

Reply:
Done